smithy-typescript-ssdk-libs/server-common/src/validation/validators.ts

```diff
@@ -90,13 +90,13 @@ export class CompositeCollectionValidator<T> implements MultiConstraintValidator
   }
 }
 
-export class CompositeMapValidator<T> implements MultiConstraintValidator<{ [key: string]: T }> {
-  private readonly referenceValidator: MultiConstraintValidator<{ [key: string]: T }>;
+export class CompositeMapValidator<T> implements MultiConstraintValidator<Record<string, T>> {
```

Comment:
I guess this is from a rebase or something in an unrelated feature?

Reply:
Yeah, once I rebase this will go away.